docs/example-react-hooks-useReducer.md

```diff
@@ -0,0 +1,119 @@
+Basic example showing how to test useReducer hook. 
+
+
+The component: changes stateprop1 from false to true which changes some text. 
+
+```
+import React, { useReducer } from 'react';
+import * as ACTIONS from '../store/actions'
+import * as Reducer from '../store/reducer'
+
+
+const TestHookReducer = (props) => {
+  const [reducerState, dispatch] = useReducer(Reducer.Reducer1, Reducer.initialState)
+
+  const dispatchActionSuccess = () => {
+    dispatch(ACTIONS.SUCCESS)
+  }
+
+  const dispatchActionFailure = () => {
+    dispatch(ACTIONS.FAILURE)
+  }
+
+
+  return (
+    <div>
+       <div>
+        {reducerState.stateprop1
+           ? <p>stateprop1 is true</p>
+           : <p>stateprop1 is false</p>}
+       </div>
+       <button onClick={dispatchActionSuccess}>
+         Dispatch Success
+       </button>
+    </div>
+  )
+}
+
+
+export default TestHookReducer;
+
+```
+
+
+our reducer: changes stateprop1 to true based on the "SUCCESS" action. 
+
+```
```

Comment:
I'd suggest changing adding js next to the backticks, so the formatter can highlight the code block:

` ```js`

same in the other code block (use `jsx` there)